src/AmbientSounds.Uwp/Strings/en-US/Resources.resw

```diff
@@ -568,4 +568,13 @@
   <data name="ComputeShader/Octagrams" xml:space="preserve">
     <value>Octagrams</value>
   </data>
+  <data name="RenderingErrorInfo.Message" xml:space="preserve">
+    <value>There was an error that caused the rendering to fail, try restarting the app. If the issue persists, please report it by sending us an email.</value>
+  </data>
+  <data name="RenderingErrorLink.Content" xml:space="preserve">
+    <value>Report the issue</value>
+  </data>
+  <data name="RenderingErrorInfo.Title" xml:space="preserve">
+    <value>Rendering error</value>
```

Comment:
Do you have reswplus installed? These changes should lead to an updated to the autogenerated resources.cs file as well, but I don't see those changes in this pr.

Reply:
https://marketplace.visualstudio.com/items?itemName=rudyhuyn.ReswPlusExtension

Reply:
Fixed in d2b6fbeee1fda34dc8ae31e67a0e550b17317ab1 👍